packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -820,4 +850,28 @@ export class MgtFileList extends MgtTemplatedComponent {
       focusedItem.scrollIntoView({ behavior: 'smooth', block: 'nearest', inline: 'start' });
     }
   }
+
+  /**
+   * Handle reload of File List
+   *
+   */
+  public reload(clearCache = false) {
+    if (clearCache) {
+      // clear cache here
+      const cacheSchema: CacheSchema = {
+        name: 'file-lists',
+        stores: {
+          fileLists: 'fileLists',
+          insightfileLists: 'insightfileLists'
+        },
+        version: 1
+      };
+
+      let cache: CacheStore<CacheFileList>;
+      cache = CacheService.getCache<CacheFileList>(cacheSchema, 'fileLists');
+      cache.clearStore();
```

Comment:
Same here - there is already a schema for filelist that can be reused.

```suggestion
      let cache: CacheStore<CacheFileList>;
      cache = CacheService.getCache<CacheFileList>(schemas.fileLists, schemas.fileLists.stores.fileLists);
      cache.clearStore();
```

Clearing the cache should also clear the cache for the insightfileList store as well